server/src/main/java/org/elasticsearch/search/SearchHit.java

```diff
@@ -587,6 +618,8 @@ public static SearchHit createFromMap(Map<String, Object> values) {
         }
         searchHit.score(get(Fields._SCORE, values, DEFAULT_SCORE));
         searchHit.version(get(Fields._VERSION, values, -1L));
+        searchHit.setSeqNo(get(Fields._SEQ_NO, values, SequenceNumbers.UNASSIGNED_SEQ_NO));
+        searchHit.setPrimaryTerm(get(Fields._PRIMARY_TERM, values, SequenceNumbers.UNASSIGNED_PRIMARY_TERM));
```

Comment:
Same here ?

Reply:
done. What tests am I missing that were supposed to catch it?

Reply:
`SearchHitTests` and `SearchHitsTests`, they are supposed to test the (de)serialization of random search hits using xcontent and streamable but you need to fill these fields in the `createTestItem` function to ensure that they are tested.